src/AxisAlignedBB.php

```diff
@@ -223,6 +223,29 @@ public function trimmedCopy(int $face, float $distance) : AxisAlignedBB{
 		return $this->extendedCopy($face, -$distance);
 	}
 
+	/**
+	 * Shifts the AABB in the given direction.
+	 * @see AxisAlignedBB::extend()
+	 *
+	 * @param float $distance Positive values shift the AABB in the given direction, negative values in the opposite.
+	 *
+	 * @return $this
+	 * @throws \InvalidArgumentException
+	 */
+	public function shift(int $face, float $distance) : AxisAlignedBB{
+		return $this->extend($face, $distance)->extend(Facing::opposite($face), -$distance);
```

Comment:
```suggestion
		return $this->extend($face, $distance)->trim(Facing::opposite($face), $distance);
```
Could have also used `AxisAlignedBB::trim()` because it might look better, but overall both approaches do the same.